beacon-chain/rpc/service.go

```diff
@@ -240,7 +240,6 @@ func NewService(ctx context.Context, cfg *Config) *Service {
 		GenesisFetcher:         s.cfg.GenesisFetcher,
 		FinalizationFetcher:    s.cfg.FinalizationFetcher,
 		TimeFetcher:            s.cfg.GenesisTimeFetcher,
-		BlockFetcher:           s.cfg.ExecutionChainService,
```

Comment:
We had 2 registrations of Block Fetcher, one as block fetcher and one as Eth1BlockFetcher